Gemfile.lock

```diff
@@ -390,4 +402,4 @@ RUBY VERSION
    ruby 3.1.2p20
 
 BUNDLED WITH
-   2.3.19
+   2.3.23
```

Comment:
@BohdaR 
Looks like the bundler version is different. Please paste the version into Readme.md

Reply:
Please use Bundler version 2.3.19